### DISCUSSION TITLE
Router

### DISCUSSION THREAD

**Discussion by pycodebuild** [bitsoftd/Accumulator-frontend#13] [General]
What type of router have you used for this project routing?

**Comment by geniusdev329** [comment]
> What type of router have you used for this project routing?

I have used Browser Router and Hash Router for this app.

**Comment by pycodebuild** [comment]
What type of css have you used for this app?
I'm confused a little. Have you used a mix of css?

**Reply by geniusdev329** [reply]
Yes, Material UI, TailwindCSS, Sass.
